01-ruby-fundamentals/02-intro-to-object-types.md

```diff
@@ -77,26 +80,26 @@ _Symbols_ are a lot like strings but they are meant to represent identifiers. _S
 "elephant".object_id # => 70359726563680
 ```
 
-## Working with numbers: `Fixnum` & `Float`
+## Working with numbers: `Integer` & `Float`
 
-Working with numbers in Ruby comes in two forms. The first form represents integers (whole numbers without a decimal point). These are called `Fixnum`s in Ruby. Numbers with a decimal point are represented by the `Float` object.
+Working with numbers in Ruby comes in two forms. The first one are called `Integer`s in Ruby (whole numbers). Numbers with a decimal point are represented by the `Float` object.
```

Comment:
The updated sentence doesn't quite read correctly... no sure how to change it exactly